<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesTab</class>
 <widget class="QWidget" name="PreferencesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>This are the preferences for Abigail IDE</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Text Area</string>
       </property>
       <widget class="QCheckBox" name="columnLineCheck">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>25</y>
          <width>275</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Display a Vertical Line in te Text Area</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>182</width>
          <height>17</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Column to Display the Line:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="columnLineBox">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>55</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Font Size:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="fontSizeBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>95</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Utilities</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QCheckBox" name="lineWrapCheck">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>25</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Line Wrap by Default</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="centerOnScrollCheck">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>53</y>
          <width>140</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Center On Scroll</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>81</y>
          <width>78</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Tab Length:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="tabLengthBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>76</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
